src/cljs/harja/views/urakka/yllapitokohteet/paikkaukset/paikkaukset_toteumat.cljs

```diff
@@ -29,6 +29,7 @@
             [harja.views.urakka.yllapitokohteet.paikkaukset.paikkaukset-yhteinen :as yhteinen-view]
             [harja.views.urakka.yllapitokohteet.paikkaukset.paikkaukset-toteumalomake :as v-toteumalomake]
             [harja.ui.debug :as debug]
+            [harja.ui.dom :as dom]
```

Comment:
Voisitko tästäkin vähän siivoilla näitä: **<! timeout, [atom]**, **harja.loki**, **yllapitokohteet**, **debug**, **kentat**, **valinnat,** **komm**

Danke